test/fast-list-edit-test.js

```diff
@@ -20,7 +20,6 @@ suite('FastList > Edit >', function() {
 
   setup(function() {
```

Comment:
the number of items that get "tweaked" during all the re-ordering work is directly impacting by the pre-rendering window.
so there's probably a lot of failures coming from that.